### ISSUE TITLE
Name must be a string

### ISSUE THREAD

**Issue by valeriansaliou** [andris9/smtp-server#21]
Hello again,

Got a random error using latest v1.5.2. I don't know how to reproduce since it has been reported by my log manager, and may come from a random user using a random SMTP server trying to send a mail to my inbound mailer (which is running smtp-server).

Hope the trace is enough to fix it on your side.

```
Error: Name must be a string
    at Object.query [as reverse] (dns.js:220:13)
    at SMTPConnection.connectionReady (/srv/data_crisp/svc/crisp-relay-mailer-inbound/node_modules/smtp-server/lib/smtp-connection.js:107:9)
    at Timer.listOnTimeout (timers.js:89:15)
```

Ping me if you need further tests on my side, or debugging on my side.

Cheers :tada:

**Comment by andris9** [comment]
Published updated version as v1.6.0

**Comment by valeriansaliou** [comment]
Thanks, @andris9 :)
